src/net.cpp

```diff
@@ -45,6 +45,8 @@
 
 // We add a random period time (0 to 1 seconds) to feeler connections to prevent synchronization.
 #define FEELER_SLEEP_WINDOW 1
+/** Run the feeler connection loop once every 2 minutes or 120 seconds. **/
+static const int FEELER_INTERVAL = 60;
```

Comment:
We have two sources of randomness:

1. One FEELER_SLEEP_WINDOW is to protect against synchronization and must be random ([see discussion on feeler pull request](https://github.com/bitcoin/bitcoin/pull/8282#discussion_r73604140) and discussion in [design statement of feeler connections under risk mitigation](https://github.com/bitcoin/bitcoin/pull/8282#issue-162739771),

2. The other FEELER_INTERVAL is to rate limit feeler connections but @sipa [proposed](https://github.com/bitcoin/bitcoin/pull/8282#discussion_r73385847) it be random as well to limit practicability.

You didn't address my criticism about only moving FEELER_INTERVAL and not any of the other intervals, what does deleting FEELER_SLEEP_WINDOW have to do with that?